src/main/telemetry/hott.c

```diff
@@ -68,32 +68,72 @@
 #include "build/debug.h"
 
 #include "common/axis.h"
+#include "common/printf.h"
 #include "common/time.h"
 
 #include "drivers/serial.h"
 #include "drivers/time.h"
+#include "drivers/vtx_common.h"
 
 #include "fc/runtime_config.h"
+#include "fc/config.h"
+#include "fc/controlrate_profile.h"
 
 #include "flight/position.h"
 #include "flight/pid.h"
 
+#include "interface/settings.h"
+
 #include "io/gps.h"
+#include "io/vtx_control.h"
+#include "io/vtx.h"
+#include "io/vtx_string.h"
 
 #include "sensors/battery.h"
 #include "sensors/barometer.h"
 #include "sensors/sensors.h"
+#include "sensors/gyro.h"
 
 #include "telemetry/hott.h"
 #include "telemetry/telemetry.h"
 
 //#define HOTT_DEBUG
 
-#define HOTT_MESSAGE_PREPARATION_FREQUENCY_5_HZ ((1000 * 1000) / 5)
-#define HOTT_RX_SCHEDULE 4000
-#define HOTT_TX_DELAY_US 3000
+#define HOTT_MESSAGE_PREPARATION_FREQUENCY_5_HZ ((1000 * 1000) / 5) // 200ms
+#define HOTT_RX_SCHEDULE 5000  										// 5ms
+#define HOTT_TX_DELAY_US 1000  										// 1ms
 #define MILLISECONDS_IN_A_SECOND 1000
 
+static const char * const TableDtermLowpassType[] = {
+    "PT1",
+    "BIQUAD",
+//#if defined(USE_FIR_FILTER_DENOISE)
+    "FIR"
+//#endif
+};
+
+static const char * const vtxRTC6705Po[] = {	// RTC6705 RF Power index "---", 25 or 200 mW
+    "---",
```

Comment:
These tables are all already defined in other parts of the firmware. Please don't copy them.